tests/Command/LintTemplatesCommand/LintTemplatesCommandTest.php

```diff
@@ -89,6 +89,8 @@ public static function provideInvalidTemplates(): iterable
                 '[families][missing_attribute_negative_allowed_family][attributes][1][negative_allowed] This field is missing.',
                 '[families][wrong_attribute_validation_rule_family][attributes][3][validation_rule] This value is not a valid validation rule. Please use one of the following : url,email.',
                 '[families][missing_attribute_validation_rule_family][attributes][1][validation_rule] This field is missing.',
+                '[families][missing_attribute_group_family][attributes][1][group] This field is missing.',
+                '[families][wrong_attribute_group_family][attributes][1][group] This attribute group does not exist.'
             ]
         ];
         yield 'Invalid attribute options' => [
```

Comment:
It misses all tests case about attribute groups file validation. You can take example on the industry test case line 37, it is almost the same tests cases